server/models/actionsModel.js

```diff
@@ -5,6 +5,7 @@ function actionsModel (actions, errorMessage) {
     return {
       heading: `${action.id}: ${action.description}`,
       actionID: action.id,
+      preCheck: action['pre-check'],
```

Comment:
In the interests of simplicity, I've renamed the property `precheck` in the backend process and data.

Reply:
Done